HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -297,12 +298,25 @@
     <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:BuildingSummary/h:Site'>
       <sch:assert role='ERROR' test='count(h:SiteType) &lt;= 1'>Expected 0 or 1 element(s) for xpath: SiteType</sch:assert>
       <sch:assert role='ERROR' test='count(h:ShieldingofHome) &lt;= 1'>Expected 0 or 1 element(s) for xpath: ShieldingofHome</sch:assert>
-      <sch:assert role='ERROR' test='count(h:extension/h:GroundConductivity) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/GroundConductivity</sch:assert>
-      <sch:assert role='ERROR' test='number(h:extension/h:GroundConductivity) &gt; 0 or not(h:extension/h:GroundConductivity)'>Expected extension/GroundConductivity to be greater than 0</sch:assert>
+      <sch:assert role='ERROR' test='count(h:Soil) &lt;= 1'>Expected 0 or 1 element(s) for xpath: Soil</sch:assert> <!-- See [Soil] -->
```

Comment:
This is a breaking change, need to call it out in the changelog.

Since we were previously using an extension element, a software tool that continues to write that element won't get any errors but the value would be ignored, which is not a good outcome. So let's also mark the old field as deprecated (like I did [here](https://github.com/NREL/OpenStudio-HPXML/blob/4fd2701e4f70734a17dd7bd3306f351d1f52e19f/HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml#L346), for example).